dev/vagrant/conf/README.md

```diff
@@ -10,7 +10,7 @@ There are two layers of configuration built into the box.
 You may add a `pre_ansible.sh` script file to this directory to run any alterations **before** ansible bootstrap procedure will run.
 
 This might come handy if you need to change or replace that bootstrap process. For example you replacing the entire `/etc/ansible` directory.
-For most cases, you are probably looking to add configuration in [pre/poststart bootstrap](pre_bootstrap/README.md)
+For most cases, you are probably looking to add configuration in [pre/poststart bootstrap](../pre_bootstrap/README.md)
```

Comment:
```suggestion
For most cases, you are probably looking to add configuration in [pre/pre_bootstrap](../pre_bootstrap/README.md)
```

Reply:
These should probably  be fixes in the template aswell